lib/spec/runners/operations.ex

```diff
@@ -97,18 +97,20 @@ defmodule OperationsTestRunner do
       YamlElixir.read_from_file!(case_dir <> "/execution.yaml")
       |> SpecTestUtils.sanitize_yaml()
 
-    status = if execution_valid, do: :valid, else: :invalid
+    _status = if execution_valid, do: :valid, else: :invalid
 
-    result =
-      Operations.process_execution_payload(pre, body, fn _payload -> {:ok, status} end)
```

Comment:
Doesn't this change the behaviour? Why do these tests pass?

Reply:
What do you mean? There were some tests that return invalid from the execution client (trivial tests tbh) that we can't have anymore because we moved the call to another place in the code.

So we filter those tests out